KIF Tests/OffscreenTests.m

```diff
@@ -0,0 +1,36 @@
+//
+//  OffscreenTests.m
+//  KIF Tests
+//
+//  Created by Steve Sun on 2023-03-28.
+//
+
+#import <KIF/KIF.h>
+#import "KIFTestStepValidation.h"
+
+@interface OffscreenTests : KIFTestCase
+@end
+
+@implementation OffscreenTests
+
+- (void)beforeEach
+{
+    [tester tapViewWithAccessibilityLabel:@"Offscreen Views"];
+}
+
+- (void)afterEach
+{
+    [tester tapViewWithAccessibilityLabel:@"Test Suite" traits:UIAccessibilityTraitButton];
+}
+
+- (void)testViewOffscreen
```

Comment:
We should probably move the views back into the scrollable region and unhide them, and make sure that they pass the `waitForView` checks. While they're offscreen, they we should probably be able to ensure that they fail the `waitForView` checks.